view/frontend/layout/catalog_product_view.xml

```diff
@@ -21,5 +21,10 @@
         </arguments>
       </block>
     </referenceContainer>
+    <referenceContainer name="before.body.end">
+      <block class="Magento\Framework\View\Element\Template"
+        template="IMI_FriendlyCaptcha::imi_friendly_captcha_js.phtml" name="friendly_captcha_js"
+        ifconfig="imi_friendly_captcha/frontend/enabled"/>
```

Comment:
don't we have a setting for the product reviews and should only add the JS if that specific setting is on?